src/SearchSystem/PatronSearch/PatronPerformSearch.cs

```diff
@@ -0,0 +1,11 @@
+using userSystem;
+using userSystem.Concrete;
+
+namespace SearchSystem;
+public class PatronPerformSearch{
```

Comment:
It looks good but why don't you make use of an interface for this class and also for `BookPerformSearch`? I see we have the same method and firm in both classes?